de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports/5454790_sys_FRESH-1010_fix_docs_sales_inout_details_hu_individual.sql

```diff
@@ -0,0 +1,56 @@
+
+DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.Docs_Sales_InOut_Details_HU_Individual ( IN Record_ID numeric, IN AD_Language Character Varying (6) );
+
+
+-- table of same sort as  de_metas_endcustomer_fresh_reports.Docs_Sales_InOut_Details_HU
+DROP TABLE IF EXISTS de_metas_endcustomer_fresh_reports.Docs_Sales_InOut_Details_HU_Individual;
+
+CREATE TABLE de_metas_endcustomer_fresh_reports.Docs_Sales_InOut_Details_HU_Individual
```

Comment:
did you also consider not creating this physical table, but declaring it in the function's `RETURNS` part?
if you did and decided against, still please use `COMMENT ON` for the table and the function to document which issue thsi is about and roughly what the function does.